build.Tests.ps1

```diff
@@ -0,0 +1,60 @@
+$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+$sut = (Split-Path -Leaf $MyInvocation.MyCommand.Path) -replace '\.Tests\.', '.'
+# . "$here\$sut"
+$script = "$here\$sut"
+
+& $script -ErrorAction Stop -PowerShellModuleName PowerShellGet -PowerShellModuleVersion "2.1.3"
+
+Describe $script {
+  It "creates a module" {
+    Test-Path "import/powershellget/metadata.json" | Should -BeTrue
+  }
+
+  It "has a REFERENCE.md" {
+    Test-Path "import/powershellget/REFERENCE.md" | Should -BeTrue
+  }
+
+  It "has a type generated" {
+    Test-Path "import/powershellget/lib/puppet/type/dsc_psmodule.rb" | Should -BeTrue
+  }
+
+  Context "inside the module" {
+    BeforeAll { Push-Location -Path "import/powershellget" }
+    AfterAll { Pop-Location }
+
+    It '`pdk validate` runs succesfully' -TestCases $validators {
+      # reports, but does not print errors :-(
+      # TODO: fix to show detailed errors
+      pdk validate 'metadata,puppet,ruby,tasks,yaml'
+      $LASTEXITCODE | Should -Be 0
+    }
+    It "is buildable" {
+      pdk build
+      $LASTEXITCODE | Should -Be 0
+    }
+    It "is installable" {
+      pdk bundle exec puppet module install --verbose pkg/*.tar.gz
+      $LASTEXITCODE | Should -Be 0
+    }
+  }
+
+  Context "when passing in invalid values" {
+    It "reports the error" {
+      New-PuppetDscModule -PowerShellModuleName "____DoesNotExist____" -ErrorAction Stop | Should -Throw
```

Comment:
I _think_ the statement to the left of the pipe needs to be wrapped in `{}` to evaluate correctly without introducing an actual error in the test run.

Reply:
yeah, that worked